leftwm-core/src/models/screen.rs

```diff
@@ -100,6 +100,20 @@ impl From<x11_dl::xrandr::XRRCrtcInfo> for Screen {
     }
 }
 
+impl From<x11rb::protocol::randr::GetCrtcInfoReply> for Screen {
```

Comment:
Couldn't it be better to implement `Into<T>` from the backends direclty ? I feel like this would be more natural